workflow-ui/core-android/src/test/java/com/squareup/workflow1/ui/WorkflowLayoutTest.kt

```diff
@@ -0,0 +1,40 @@
+package com.squareup.workflow1.ui
+
+import android.content.Context
+import android.os.Bundle
+import android.os.Parcelable
+import android.util.SparseArray
+import android.view.View
+import androidx.test.core.app.ApplicationProvider
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE, sdk = [28])
```

Comment:
worth a comment about the error you were seeing (and why it needs to be specific here)

Reply:
Done.